src/Sylius/Bundle/AddressingBundle/Resources/config/validation/Province.xml

```diff
@@ -34,7 +34,7 @@
             </constraint>
             <constraint name="Regex">
                 <option name="message">sylius.province.code.regex</option>
-                <option name="pattern">/[A-Z]{2}-[^W\d]{2,}/</option>
```

Comment:
We use https://en.wikipedia.org/wiki/ISO_3166-2 so it should always be 2 letter code